views/about.tmpl

```diff
@@ -1,3 +1,17 @@
+{{define "title"}}{{.PageContext.ShortName}} | About{{end}}
+
+{{define "open-graph"}}
+    <meta name="twitter:card" content="summary_large_image"/>
+    <meta name="twitter:site" content="@URY1350"/>
```

Comment:
Yeah that sounds good.
Open-graph requires the full URL (https://ury.org.uk/...) is it worth putting this in the config also?